src/common/services/api/cart.service.js

```diff
@@ -134,6 +135,23 @@ function cart(cortexApiService){
       path: uri
     });
   }
+
+  function giftStartDate(day){
+    return cortexApiService.get({
+      path: ['nextdrawdate'],
+      cache: true
+    }).map((data) => {
+      let drawDate = data['next-draw-date'].split('-');
+      drawDate = new Date(drawDate[0], (drawDate[1] - 1), drawDate[2]);
```

Comment:
I was thinking the same thing but how come `new Date('2014-12-01')` gives me `Sun Nov 30 2014 16:00:00 GMT-0800 (PST)` and `new Date('2014-12-02')` gives me `Mon Dec 01 2014 16:00:00 GMT-0800 (PST)`? The day seems off by one... I usually use [moment](http://momentjs.com/) for dates...